pkg/heat/dbsync.go

```diff
@@ -88,10 +88,8 @@ func DBSyncJob(
 	initContainerDetails := APIDetails{
 		ContainerImage:            instance.Spec.HeatAPI.ContainerImage,
 		DatabaseHost:              instance.Status.DatabaseHostname,
-		DatabaseUser:              instance.Spec.DatabaseUser,
```

Comment:
this is what had to be removed otherwise we get new DBSyncJob everytime username changes